drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -2177,7 +2177,11 @@ void RasterizerStorageGLES3::sky_set_texture(RID p_sky, RID p_panorama, int p_ra
 			}
 
 			glBindFramebuffer(GL_DRAW_FRAMEBUFFER, tmp_fb);
-			glFramebufferTextureLayer(GL_DRAW_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, sky->radiance, 0, lod);
+#ifdef GLES_OVER_GL
+			glFramebufferTexture(GL_DRAW_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, sky->radiance, lod);
+#else
+			glFramebufferTexture2D(GL_DRAW_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D, sky->radiance, lod);
```

Comment:
Can't we use `glFramebufferTexture2D` in both cases? The rest of the code seems to have calls to `glFramebufferTexture2D` for both GL and GLES.

Reply:
Yea. Looks like we can. I should have looked deeper.